sdk/python/feast/feature_store.py

```diff
@@ -153,6 +166,57 @@ def get_historical_features(
         )
         return job
 
+    @property
+    def project(self) -> str:
+        return "default"
+
+    def materialize(
+        self, feature_views: List[str], start_date: datetime, end_date: datetime
+    ):
+        full_feature_views = []
+        registry = self._get_registry()
+        for name in feature_views:
+            feature_view = registry.get_feature_view(name, self.project)
```

Comment:
Would `list_feature_views()` be a bit faster?

Reply:
maybe, but we'd have to repeat the logic of returning an error if one of the feature views doesn't exist; I'd prefer to do this and optimize later